src/app/content/components/BookBanner.spec.tsx

```diff
@@ -88,6 +89,67 @@ describe('BookBanner', () => {
       const component = renderer.create(<BarWrapper colorSchema='blue' up={true} />);
       expect(component.toJSON()).toMatchSnapshot();
     });
+
+    it('defaults tab indexes on banner links', () => {
+      const component = renderer.create(<BookBanner pageNode={pageNode} book={book} />);
+
+      const linkExpanded = component.root.findByProps({'data-testid': 'details-link-expanded'});
+      const linkCollapsed = component.root.findByProps({'data-testid': 'details-link-collapsed'});
+
+      expect(linkExpanded.props.tabIndex).toBe(undefined);
+      expect(linkCollapsed.props.tabIndex).toBe(-1);
+    });
+
+    it('sets tab indexes on banner links according to scroll', () => {
+      const expandedBannerNode = assertDocument().createElement('div');
+      const collapsedBannerNode = assertDocument().createElement('div');
+
+      const createNodeMock = (element: any) => {
+        const analyticsRegion = element.props['data-analytics-region'];
+
+        if (analyticsRegion === 'book-banner-expanded') {
+          return expandedBannerNode;
+        }
+        if (analyticsRegion === 'book-banner-collapsed') {
+          return collapsedBannerNode;
+        }
```

Comment:
I think you could add test ids to these elements just so when we update the analytics region these tests will not fail